src/coreclr/jit/codegencommon.cpp

```diff
@@ -6452,6 +6452,46 @@ void CodeGen::genFnProlog()
 #pragma warning(pop)
 #endif
 
+//----------------------------------------------------------------------------------
+// getBaseOffsetOfGeneratedJumpTable: get base offset of generated jump table
+//
+// Arguments:
+//    treeNode - the GT_JMPTABLE node
+//
+// Return Value:
+//    base offset to jump table
+//
+// Assumption:
+//    The current basic block in process ends with a switch statement
+//
+unsigned CodeGen::getBaseOffsetOfGeneratedJumpTable(GenTree* treeNode)
```

Comment:
Nit: the name makes it seem like the jump table has already been generated, but this function is what emits it.
I would call it `genEmitJumpTable` instead.